opc-ua-sdk/sdk-client/src/main/java/org/eclipse/milo/opcua/sdk/client/subscriptions/TransferMonitoredItemsRequest.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2019 the Eclipse Milo Authors
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+package org.eclipse.milo.opcua.sdk.client.subscriptions;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.function.BiConsumer;
+
+import org.eclipse.milo.opcua.sdk.client.api.subscriptions.UaMonitoredItem;
+import org.eclipse.milo.opcua.sdk.client.api.subscriptions.UaSubscription;
+
+public class TransferMonitoredItemsRequest {
+    private final List<MonitoredItemTransferRequest> monitoredItems;
+    private final BiConsumer<UaMonitoredItem, Integer> itemTransferCallback;
+
+    public TransferMonitoredItemsRequest() {
+        this.monitoredItems = null;
+        this.itemTransferCallback = null;
+    }
+
+    public TransferMonitoredItemsRequest(
+        List<MonitoredItemTransferRequest> monitoredItems, BiConsumer<UaMonitoredItem,
```

Comment:
Looks like autoformat did a number on here; I'm gonna put the BiConsumer on it's own newline.